src/components/CardList/Card/Card.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import s from '../Cards.module.css';
+
+export interface CardProps {
```

Comment:
use type for props
export type CardProps = { 
...
}

Reply:
I need to check why